lib/collection/response.js

```diff
@@ -123,7 +130,7 @@ _.assign(Response.prototype, /** @lends Response.prototype */ {
      * @returns {String|undefined}
      */
     text: function () {
-        return (this.stream ? util.bufferOrArrayBufferToString(this.stream) : this.body);
+        return (this.stream ? util.bufferOrArrayBufferToString(this.stream, this.mime().charset) : this.body);
```

Comment:
@shamasis mime uses **this.stream || this.body**  [here](https://github.com/postmanlabs/postman-collection/pull/232/files#diff-a2d0c3519ebfb4ceb7c93d7911ee0718L178),  so using mime() here is safe